FSJ_django20_project/FSJ/templates/FSJ/coord_student_list.html

```diff
@@ -0,0 +1,53 @@
+{% extends "FSJ/base_with_banner.html" %}
+{% load i18n %}
+
+{% block heading %}
+    Students
+{% endblock %}
+
+{% block content %}
+<head>
+<style>
+table {
+    border-collapse: collapse;
+    width: 50%;
+}
+
+td, th {
+    border: 1px solid #dddddd;
+    text-align: left;
+    padding: 8px;
+}
+
+tr:nth-child(even) {
+    background-color: #dddddd;
+}
+
+</style>
+</head>
+<body>
+
+
+{% if student_list %}
+    <div>
+    <table align = "center">
+     <tr>
+      <th>CCID</th>
+      <th>Year of Study</th>
+      <th>Program</th>
```

Comment:
There's no table header for the "view" link column, leaving a strange gap in the table.